code/modules/mob/living/simple_animal/hostile/carp.dm

```diff
@@ -96,6 +99,21 @@
 	base_dead_overlay.appearance_flags = RESET_COLOR
 	add_overlay(base_dead_overlay)
 
+/mob/living/simple_animal/hostile/carp/proc/chomp_plastic()
+	var/obj/item/storage/cans/tasty_plastic = locate(/obj/item/storage/cans) in oview(src, 9)
+	if(tasty_plastic)
+		snack_distance = get_dist(src.loc,tasty_plastic.loc)
+		if(snack_distance <= 1)
+			src.visible_message("<span class='notice'>[src] gets it's head stuck in [tasty_plastic], and chokes on it!</span>", "<span class='notice'>You try to avoid [tasty_plastic], but it looks so... delicious... OH NO IT'S CHOKING YOU!</span>")
+			new /obj/effect/decal/cleanable/plastic(src.loc)
+			adjustBruteLoss(5)

```

Comment:
I kind of feel like this should do more than 5 damage, since "OH NO IT'S CHOKING YOU" kind of implies something more serious than a mere 5 damage.

Reply:
Carp have 25 health. They instantly lose 20% of their total health as a result.

Reply:
What do space carp breathe :confused: